config/sync/system.site.yml

```diff
@@ -1,14 +1,15 @@
 uuid: a401790a-be78-414b-92a4-eaae7819c9f4
-name: 'FEMA SF-425 Prototype'
+name: 'SF-425 Processing Prototype'
 mail: admin@example.com
 slogan: ''
 page:
   403: ''
   404: ''
-  front: /dashboard
+  front: /user/login
```

Comment:
See #48 ... which redirects anons to /user/login, but let the homepage be something useful for our logged in users. (i.e. can you leave this as /dashboard?)